contract/src/main/java/network/misq/contract/Party.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package network.misq.contract;
+
+import lombok.Getter;
+import network.misq.network.p2p.NetworkId;
```

Comment:
Build complains `Party.java:21: error: package network.misq.network.p2p does not exist`

Reply:
I will have a look. I merge it with the changes of the other PRs.

Reply:
Hm... for me it works. Made a clean and full build. In IntelliJ there are issues when you add modules that it does not refelct that in the IDE. Easiest is to shutdown, delete .idea folder and let it rebuild it from the gradle module definitions.

Reply:
Merged other PRs and rebased....

Reply:
> Build complains `Party.java:21: error: package network.misq.network.p2p does not exist`

Ah seems there was a commit not pushed ;-(... Should work now.